packages/web/src/components/banner/UpdateWebAppBanner.js

```diff
@@ -0,0 +1,28 @@
+import { ReactComponent as IconArrow } from 'assets/img/iconArrowGrey.svg'
+import Banner from 'components/banner/Banner'
+import Pill from 'components/pill/Pill'
+
+import styles from './UpdateAppBanner.module.css'
+
+export const UpdateWebAppBanner = ({ onAccept, onClose }) => (
+  <Banner onClose={onClose} isElectron>
+    <div className={styles.updateAppBanner} onClick={onAccept}>
+      <div className={styles.content}>
+        <div className={styles.contentSelection}>
+          <span className={styles.celebration}>
+            <i className='emoji sparkles' />
+          </span>
+          <div className={styles.text}>A New Web Version Is Available</div>
```

Comment:
We probably shouldn't expose the fact that this is a "web" update to the user. Maybe just `A New Version Is Available`?